query/language/match.feature

```diff
@@ -2362,9 +2454,9 @@ Feature: TypeQL Match Clause
       match $phrase isa favorite-phrase;
       """
     Then uniquely identify answer concepts
-      | phrase                             |
+      | phrase                        |
       | attr:favorite-phrase:你明白了吗    |
-      | attr:favorite-phrase:בוקר טוב      |
+      | attr:favorite-phrase:בוקר טוב |
```

Comment:
Is IDEA doing this weird formatting around double width characters, or is this by hand? I keep having to fix tables like this.

Reply:
It's IDEA, I mentioned it in one of my previous PRs you reviewed. So I decided to ignore it. Sorry to hear about your struggles =(

Reply:
This one is exceptionally ugly, so I fixed it =)